07week/todolist/src/App.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+class App extends Component {
+  constructor(props){
+      super(props);
+      this.state = {
+        list: [],
+        inputValue: '',
+      }
+  }
+
+handleInputChange = (e)=>{
+  this.setState({inputValue: e.target.value})
+};
+
+handleClear = ()=>{
```

Comment:
this function isn't really doing anything right now. In your handleSubmit function you are already setting the input value back to an empty string, so this function is never even used.